guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<p>"Let's look at some examples of creating reports for some study areas. To learn more about Study Areas, refer to <code>Performing Enrichment</code> guide. "</p><ul><li>"Let's look at some examples of creating reports for selected study areas. To learn more about <code>Study Areas</code>, refer to the guide on <code>Performing Enrichment (here, link is needed)</code>. "</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>